pandas/core/reshape/reshape.py

```diff
@@ -174,25 +180,33 @@ def _indexer_and_to_sort(
         return indexer, to_sort
 
     @cache_readonly
-    def sorted_labels(self) -> list[np.ndarray]:
+    def labels(self) -> list[np.ndarray]:
         indexer, to_sort = self._indexer_and_to_sort
         if self.sort:
             return [line.take(indexer) for line in to_sort]
         return to_sort
 
-    def _make_sorted_values(self, values: np.ndarray) -> np.ndarray:
+    @cache_readonly
+    def sorted_labels(self) -> list[np.ndarray]:
         if self.sort:
-            indexer, _ = self._indexer_and_to_sort
+            return self.labels
```

Comment:
This seems a bit confusing to me: what was `sorted_labels` has become `labels`, and the code for `sorted_labels` returns the same result as `labels` when `sort=True`. If there are good reasons behind these names, maybe add a short docstring to make that clear? Otherwise, perhaps a renaming is in order.